test/multiverse/suites/stripe/stripe_instrumentation_test.rb

```diff
@@ -0,0 +1,160 @@
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+require 'json'
+require 'stripe'
+require 'net/http'
+
+class StripeInstrumentation < Minitest::Test
+  API_KEY = '123456789'
+
+  def setup
+    Stripe.api_key = API_KEY
+    # Creating a new connection and response, which both get stubbed
+    # later, helps us get around needing to provide a valid API key
+    @connection = Stripe::ConnectionManager.new
+    @response = Net::HTTPResponse.new('1.1', '200', 'OK')
+    # Bypass #stream_check ("attempt to read body out of block")
+    @response.instance_variable_set(:@read, true)
+    @response.body = {
+      object: 'list',
+      data: [{'id': '12134'}],
+      has_more: false,
+      url: '/v1/charges'
+    }.to_json
+  end
+
+  def test_version_supported
+    assert(Stripe::VERSION >= '5.38.0')
```

Comment:
Does this need `Gem::Version.new` to compare the values?

Reply:
We were running fine without it, but I've definitely run into issues with this in the past. To be safe and match instrumentation, I've updated https://github.com/newrelic/newrelic-ruby-agent/pull/2180/commits/c997c511306d9a07fb8f68c839f653cdc3dd3cc3